examples/natmod/README.md

```diff
@@ -67,8 +67,8 @@ with `pip`.
 Each example provides a Makefile. You should specify the `ARCH` argument to
 make (one of x86, x64, armv6m, armv7m, xtensa, xtensawin):
 
-```
-$ cd features0
-$ make ARCH=armv7m
-$ mpremote cp features0.mpy :
+```text
+cd features0
+make ARCH=armv7m
+mpremote cp features0.mpy :
```

Comment:
How about just making this a block indented by 4 spaces, without the three ticks?

Also, prefixing with dollar signs matches all the other README.md files, so I'd prefer to keep them.  They indicate you need to use the REPL.

Reply:
> How about just making this a block indented by 4 spaces, without the three ticks?

Can do but that it raises [MD046](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md046---code-block-style). I personally prefer the 'fencing' style (with backticks) because the language can be specified and so fencing can be used consistently, everwhere. But, it's up to you!

> Also, prefixing with dollar signs matches all the other README.md files, so I'd prefer to keep them. They indicate you need to use the REPL.

This raises [MD014](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md014---dollar-signs-used-before-commands-without-showing-output) and there's some justification for it. Probably the biggest issue is that copy/paste is more useful without the '$'. But again, up to you. 😄